doc/attr.xml

```diff
@@ -1211,3 +1211,60 @@ gap> OutNeighbours(sym);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="UndirectedSpanningTree">
+<ManSection>
+  <Attr Name="UndirectedSpanningTree" Arg="digraph"/>
+  <Attr Name="UndirectedSpanningForest" Arg="digraph"/>
+  <Returns>A digraph, or <K>fail</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is a digraph with at least one vertex, then
+    <C>UndirectedSpanningForest</C> returns an undirected spanning forest of
+    <A>digraph</A>, otherwise this attribute returns <K>fail</K>. See <Ref
+      Oper="IsUndirectedSpanningForest" /> for the definition of an undirected
+    spanning forest.
+
+    <P/>
+
+    If <A>digraph</A> is a digraph with at least one vertex and whose <Ref
+      Attr="MaximalSymmetricSubdigraph"/> is connected (see <Ref
+      Prop="IsConnectedDigraph" />), then <C>UndirectedSpanningTree</C> returns
+    an undirected spanning tree of <A>digraph</A>, otherwise this attribute
+    returns <K>fail</K>.  See <Ref Oper="IsUndirectedSpanningTree" /> for the
+    definition of an undirected spanning tree.
+
+    <P/>
+
+    Note that for a digraph that has an undirected spanning tree, the attribute
+    <C>UndirectedSpanningTree</C> returns the same digraph as does the attribute
+    <C>UndirectedSpanningForest</C>.
+
+    <P/>
```

Comment:
Yes, it's redundant in the sense that it doesn't affect the output. But semantically I put it there since it still marks the end of the preceding paragraph. I'll remove it.